### PR TITLE
Feature/audittrail admin inline

### PR DESCRIPTION
**Pull request by annashamray**
Issue - https://github.com/open-zaak/open-zaak/issues/245

### CODE REVIEW

File: src/openzaak/components/documenten/admin.py [hunk 1]
```diff
@@ -40,6 +46,9 @@ def get_not_lock_display(self, obj) -> bool:
     get_not_lock_display.short_description = "free to change"
     get_not_lock_display.boolean = True
 
+    def get_viewset(self, request):
```
Comment (joeribekker) [inline src/openzaak/components/documenten/admin.py]: Why is this here?

File: src/openzaak/utils/admin.py [hunk 2]
```diff
@@ -223,9 +227,75 @@ def delete_model(self, request, obj):
...
+        # add new
+        for obj in formset.new_objects:
+            data_after = self.get_serializer_data(request, viewset, obj)
+            self.trail(
+                obj, viewset, request, CommonResourceAction.create, None, data_after
+            )
+
+
+class AuditTrailInlineAdminMixin(object):
+    viewset = None
```
Comment (joeribekker) [inline src/openzaak/utils/admin.py]: Maybe we should add a check if viewset is still None on init and raise an ImproperlyConfigured exception to prevent accidental left outs.

File: src/openzaak/components/documenten/admin.py (hunk 1 above)
Comment (annashamray) [reply src/openzaak/components/documenten/admin.py]: `EnkelvoudigInformatieObjectCanonical` doesn't have matching viewset, so the changes in the admin can't be logged. But in the admin page it has inline objects - `EnkelvoudigInformatieObject` and `Gebruiksrechten` which have related viewsets with AuditTrail. So we can log only changes in the inline objects here (it's a special case).

If we don't override `get_viewset` method the exception will be raised since this class doesn't have `viewset` attribute.

File: src/openzaak/utils/admin.py (hunk 2 above)
Comment (annashamray) [reply src/openzaak/utils/admin.py]: In this case we couldn't trail changes in objects with no related viewset. Like EnkelvoudigInformatieObjectCanonical
